src/consentManager.ts

```diff
@@ -168,3 +168,23 @@ export const BackendSyncOption = makeEnum({
 /** Override type */
 export type BackendSyncOption =
   typeof BackendSyncOption[keyof typeof BackendSyncOption];
+
+/**
+ * Whether or not to run cross-domain sync
```

Comment:
```suggestion
 * Whether or not to run local on-device same-site cross-domain sync
```
Remote sync is also cross-domain

Reply:
I updated the comment, but unsure if I fully understand the `on-device same-site` portion. can you elaborate? @eligrey

Reply:
Local sync is about local (non-remote AKA on-device) sync.

It's local because the synchronization mechanism doesn't require an external connection

Reply:
Ohh for some reason I didn't connect the device piece. thanks!!